File: KCD/HMMasterMapInfoCommand.m
//
//  HMMasterMapInfoCommand.m
//  KCD
//
//  Created by Hori,Masaki on 2014/02/16.
//  Copyright (c) 2014年 Hori,Masaki. All rights reserved.
//

#import "HMMasterMapInfoCommand.h"

@implementation HMMasterMapInfoCommand
- (NSString *)dataKey
{
	return @"api_data_mst_mapinfo";
}
- (void)execute
{
	[self commitJSONToEntityNamed:@"MasterMapInfo"];
}
@end

File: KCD/HMMasterMapInfoCommand.swift
//
//  HMMasterMapInfoCommand.swift
//  KCD
//
//  Created by Hori,Masaki on 2015/01/11.
//  Copyright (c) 2015年 Hori,Masaki. All rights reserved.
//

import Cocoa

class HMMasterMapInfoCommand: HMJSONCommand {
	override var dataKey: String {
		return "api_data.api_mst_mapinfo"
	}
	override func execute() {
		super.commitJSONToEntityNamed("MasterMapInfo")
	}
}
